### DISCUSSION TITLE
I couldn't pull docker image from GitHub Container Registry

### DISCUSSION THREAD

**Discussion by rrrrayyyy** [community/community#25426] [Packages]
<p>I published my image on GitHub Container Registry and it is public now.</p>
<p>And then I ran <code>docker pull</code> and got this error:</p>
<pre><code class="lang-auto">~ &gt; docker pull ghcr.io/resotto/goilerplate-pg:latest
Error response from daemon: Get https://ghcr.io/v2/resotto/goilerplate-pg/manifests/latest: denied: denied
</code></pre>
<p>How can I pull this image?</p>

**Comment by airtower-luna** [comment]
<p>Odd, that command looks correct, and I was just able to download the manifest without issue. If you just now made the image public, maybe the setting just took a little longer than expected to become active?</p>

**Comment by rrrrayyyy** [comment]
<p>Thank you for your replying.</p>
<p>I made this image public some time ago so it might be not time problem, I think.</p>
<p>So this is other problem??</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Well, there’s one more thing I can think of: The download I did was anonymous. If you’re logged in (using <code>docker login</code>), check if the token you’re using has the “read:packages” scope. If it doesn’t, you need to either update the token so it has the scope, or try logging out (<code>docker logout ghcr.io</code>) and doing an anonymous pull.</p>

**Comment by rrrrayyyy** [comment]
<p>After running <code>docker logout</code>, I could do that.</p>
<p>Thank you so much.</p>
